pkg/local_object_storage/metabase/put.go

```diff
@@ -448,9 +448,9 @@ func splitInfoFromObject(obj *object.Object) (*objectSDK.SplitInfo, error) {
 	return info, nil
 }
 
-// mergeSplitInfo ignores conflicts and rewrites `to` with non empty values
+// MergeSplitInfo ignores conflicts and rewrites `to` with non empty values
 // from `from`.
-func mergeSplitInfo(from, to *objectSDK.SplitInfo) *objectSDK.SplitInfo {
+func MergeSplitInfo(from, to *objectSDK.SplitInfo) *objectSDK.SplitInfo {
```

Comment:
that func started to be used somewhere in external pkg, if i understand everything right. shouldn't it be moved somewhere in more general pkg(`util`?), or this is always used with `put` context?

Reply:
You are right, it can be moved outside of metabase package. It is deeply connected with internal storage engine operations, so I think it should stay at `pkg/local_object_storage/common`, for example.

Reply:
sorry, yes, i was talking about `pkg/local_object_storage/util` not about `pkg/util`

Reply:
Fixed.